.yamato/project.metafile

```diff
@@ -45,9 +45,9 @@ projects:
       - 2021.2
       - 2020.3
       - trunk
-#  - name: testproject-tools-integration
-#    path: testproject-tools-integration
-#    validate: false
-#    test_editors:
-#      - 2021.2
-#      - trunk
+  - name: testproject-tools-integration
+    path: testproject-tools-integration
+    validate: false
+    test_editors:
+      - 2021.2
+      - trunk
```

Comment:
this looks out of scope in this PR

Reply:
I asked for this - this was turned off recently because tests were unstable due to missing fixes from experimental-develop, and this refactor/backport in theory was supposed to fix it

Reply:
nope, those changes got reverted entirely by #1743

Reply:
can you elaborate? I don't see the relationship with that PR

I'm not opposed to re-enabling the tools integration tests into a separate PR though

Reply:
nope I'm stupid, I misread these as multiprocess for some reason.
but still looks out of scope, there's too much going on under this PR.